tests/modules/images/auto-sizes/auto-sizes-test.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Tests for the Auto-sizes for Lazy-loaded Images module.
+ *
+ * @package performance-lab
+ * @group   auto-sizes
+ */
+
+class AutoSizesTests extends WP_UnitTestCase {
+
+	/**
+	 * Attachment ID.
+     *
```

Comment:
I realize you can't apply this suggestion here, but this will fix the linting error I believe.

```suggestion
	  *
```

Reply:
Yep! Handled in https://github.com/WordPress/performance/pull/931, which also requires some extra hoops to add the correct labels, etc.

Interestingly our `npm run lint-php` script does not catch this, while `composer run lint` does. I'm going to investigate why that is at a later date. I'm not sure what changed upstream that caused the error to show up here in the first place but it's good that it did.